### ISSUE TITLE
Problems during the reproduction process

### ISSUE THREAD

**Issue by lj-xh** [yinguanchun/SKCDF#4]
Hello, author. During my experiments using the preprocessed Synapse dataset you provided, I printed the input image dimensions and found that they are [4, 1, 64, 128, 128], which is different from the 80×160×160 size mentioned in the paper. As a result, the reproduced performance only reached Dice 55.84 and ASD 12.77. I would like to confirm whether the uploaded preprocessed dataset is correct, and whether the image size discrepancy could have affected my reproduction results. I would greatly appreciate it if you could kindly check this issue, as it plays a crucial role in my research. Many thanks in advance!

<img width="930" height="351" alt="Image" src="https://github.com/user-attachments/assets/a96894f6-96a8-4e2b-ae20-3d7af83bc8da" />

<img width="510" height="134" alt="Image" src="https://github.com/user-attachments/assets/2184138f-3b6a-4264-baa5-1b40e287cd4b" />

**Comment by yinguanchun** [comment]
> Hello, author. During my experiments using the preprocessed Synapse dataset you provided, I printed the input image dimensions and found that they are [4, 1, 64, 128, 128], which is different from the 80×160×160 size mentioned in the paper. As a result, the reproduced performance only reached Dice 55.84 and ASD 12.77. I would like to confirm whether the uploaded preprocessed dataset is correct, and whether the image size discrepancy could have affected my reproduction results. I would greatly appreciate it if you could kindly check this issue, as it plays a crucial role in my research. Many thanks in advance!
> 
> <img alt="Image" width="930" height="351" src="https://private-user-images.githubusercontent.com/186469141/507401284-a96894f6-96a8-4e2b-ae20-3d7af83bc8da.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjIyNzUyNzEsIm5iZiI6MTc2MjI3NDk3MSwicGF0aCI6Ii8xODY0NjkxNDEvNTA3NDAxMjg0LWE5Njg5NGY2LTk2YTgtNGUyYi1hZTIwLTNkN2FmODNiYzhkYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTEwNFQxNjQ5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kYWE1MDE3YmM3MTYyNWE4MjMyYmQ4OThmOTQ4YWZjODFhOTNiN2JhYjJhZWFlY2JhNTE4OGMzNjNlODE0YmI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.dWgGQxSlWAlABwihodx3LhtdNrNbt5DBhbYZYkYUZfk"> <img alt="Image" width="510" height="134" src="https://private-user-images.githubusercontent.com/186469141/507401615-2184138f-3b6a-4264-baa5-1b40e287cd4b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjIyNzUyNzEsIm5iZiI6MTc2MjI3NDk3MSwicGF0aCI6Ii8xODY0NjkxNDEvNTA3NDAxNjE1LTIxODQxMzhmLTNiNmEtNDI2NC1iYWE1LTFiNDBlMjg3Y2Q0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTEwNFQxNjQ5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kODVkOTYxYzk2ZDcyODYxNjJjYjdmYWI5ZmNhYTI5ZTFhZjdiMzg0MGUwMzM0ZmQ0Mjc3YWI5YmZiYjI4NWFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.j8zP_jhCZ0hYr5qFpm1oev3cPFCEfw1w7eB3835Ta5g">
The original size of the data we provided should be 80×160×160. 64×128×128 is the size after random clipping by data_loader.py. You can use np.load() to check the size of the data we provided. Regarding the reproduction of the results, may I ask if you used the data splits we provided? Due to the small sample size, data splits hava a relatively obvious impact on the results. When training on the Synapse dataset, the learning rate is set to 0.3. Also, I noticed that the Dice of one of  organs is 0. You can check the training log to see if this organ has consistently shown 0 on the validation set during the training process? How is the overall performance on the validation set?

**Comment by lj-xh** [comment]
Hello, and thank you for your response. After carefully reviewing my training logs, I found that the RAG performed poorly throughout the entire training process. Although I achieved the best Dice score at epoch 1404, this organ still had a score of 0 on all images during the testing phase. Do you know what might be causing this issue? Thank you very much.

<img width="1530" height="810" alt="Image" src="https://github.com/user-attachments/assets/456e33fd-3261-449c-be39-e1e544c0cb4c" />
<img width="693" height="807" alt="Image" src="https://github.com/user-attachments/assets/a8bc3bae-a32a-4bbf-992a-d3c208f44848" />

**Comment by KISGP** [comment]
@lj-xh Did you solve it? I'm having the same problem.😥
